### DISCUSSION TITLE
Should we release 0.70.9?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#62] [Patches]
Current Release: [0.70.8](https://github.com/facebook/react-native/releases/tag/v0.70.8)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by kelset** [comment]
From @mikeduminy in [prev discussion](https://github.com/reactwg/react-native-releases/discussions/56#discussioncomment-5122392):

> Hey there, there is an issue in Hermes that prevents one from loading large, complex bundles in development. The change is to simply increase the max size of the stack.
> The change hasn't been merged yet but it targets the stable branch within Hermes for 0.70. My guess is some coordination needs to happen to first merge this then bump this in RN. I'm happy to help where ever needed.

https://github.com/facebook/hermes/pull/924

**Reply by lunaleaps** [reply]
Trying to play catch up here, is the take-away from this [comment](https://github.com/facebook/hermes/issues/135#issuecomment-1448503234) that we need to pick the following:
* https://github.com/facebook/hermes/commit/03f2dffc1d0ef8b2360a6790ad425ce4013e4de3
* https://github.com/facebook/hermes/commit/1b759f40bd2f6bb72b2a353f0d9856fcbdbb981c

onto Hermes `rn/0.70-stable`? And then publish a new Hermes tag, and update it on this branch via `./scripts/hermes/bump-hermes-version.js ...` for this patch?

If yes, here's the PR of the picks for Hermes: https://github.com/facebook/hermes/pull/969

**Reply by kelset** [reply]
@lunaleaps yeah good call - thanks for doing the proper check. Yes those are the two commits, thanks for setting that PR up.

I think you can go ahead and merge, then trigger the Hermes workflow https://github.com/facebook/hermes/actions/workflows/create-tag.yml using as commit reference the hash of the last commit (after having merged the PR) in the Hermes https://github.com/facebook/hermes/commits/rn/0.70-stable branch. That should generate a new Hermes tag along the lines of `hermes-2023-04-XX-RNv0.70.8-<hash>`  (similar to https://github.com/facebook/hermes/releases/tag/hermes-2022-09-14-RNv0.70.1-2a6b111ab289b55d7b78b5fdf105f466ba270fd7, basically).

And once that one is generated, we can go in the 0.70 branch in RN core: https://github.com/facebook/react-native/tree/0.70-stable and run the `./scripts/hermes/bump-hermes-version.js -t <the_hermes_tag>` with the newly generated tag, and push that commit.

**Reply by lunaleaps** [reply]
done! Thanks for the detailed instructions!

**Comment by kelset** [comment]
once we tested them in 0.72's next RC (RC1) - let's cherry pick @NickGerleman's fixes for TextInput (EditText) (in order):
* https://github.com/facebook/react-native/commit/1743dd7ab40998c4d3491e3b2c56c531daf5dc47
* https://github.com/facebook/react-native/commit/b9e2627d1ce64e8e293e90f743bf13a515db3b4d
* https://github.com/facebook/react-native/commit/cc0ba57ea42d876155b2fd7d9ee78604ff8aa57a
* https://github.com/facebook/react-native/commit/8c9c8ba5adb59f7f891a5307a0bce7200dd3ac7d
* https://github.com/facebook/react-native/commit/0869ea29db6a4ca20b9043d592a2233ae1a0e7a2
* https://github.com/facebook/react-native/commit/5791cf1f7b43aed1d98cad7bcc272d97ab659111
* https://github.com/facebook/react-native/commit/104cb7f81ecbb81841070078c55ddf5d8ea9d24b
* https://github.com/facebook/react-native/commit/b384bb613bf533aebf3271ba335c61946fcd3303
* https://github.com/facebook/react-native/commit/92b898149956a301a44f99019f5c7500335c5553

And before that, this commit: https://github.com/facebook/react-native/commit/8a0fe30591e21b90a3481c1ef3eeadd4b592f3ed

**Reply by kelset** [reply]
PR: https://github.com/facebook/react-native/pull/36875

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/b07cf33279844d30920d8d46edb48b10ed5a1614 to address performance regressions
